beacon-chain/sync/subscriber.go

```diff
@@ -118,7 +118,7 @@ func (s *Service) registerSubscribers(epoch primitives.Epoch, digest [4]byte) {
 		s.attesterSubnetIndices,
 	)
 	// Altair Fork Version
-	if epoch >= params.BeaconConfig().AltairForkEpoch {
+	if params.BeaconConfig().AltairForkEpoch <= epoch {
```

Comment:
I did this on purpose, because with peerDAS I'll had something like:
```go
// New Gossip Topic in Deneb
if params.BeaconConfig().DenebForkEpoch <= epoch && epoch < params.BeaconConfig().FuluForkEpoch {
...
```

Reply:
So in this case, I fill cleaner to write all inequalities in the same order.

Reply:
That seems like an argument to fix peerdas rather than the other way round. It is more intuitive as it reads as:
```
Epoch is greater than or equal to fork epoch
```
rather than

```
Fork Epoch is smaller than or equal to epoch
```

Since epoch is the main argument here, it should be the first value in the conditional